src/client/Microsoft.Identity.Client/Http/HttpManager.cs

```diff
@@ -54,8 +73,20 @@ public async Task<HttpResponse> SendRequestAsync(
             X509Certificate2 bindingCertificate,
             Func<HttpRequestMessage, X509Certificate2, X509Chain, SslPolicyErrors, bool> validateServerCert,
             CancellationToken cancellationToken,
+            IRetryPolicy retryPolicy = null,
```

Comment:
Let's please not use default parameters with this one. THis will force other MSAL developers to think about retries, instead of using the "default" which is no-retry.

Reply:
Shouldn't the default retry by the LinearRetryPolicy? That's how it was already set up, for STS and MI requests.

Reply:
But we are changing the MI requests to use different policies right?

Reply:
done